### ISSUE TITLE
yafti doesn't autostart on sericea images

### ISSUE THREAD

**Issue by qoijjj** [ublue-os/startingpoint#206]
To reproduce:

rebase to a ublue sericea image

Expected behavior:

yafti autostarts

Actual behavior:

yafti doesn't appear

**Comment by xynydev** [comment]
Sway doesn't autostart programs in the same way full-blown DEs do and that's why the desktop file doesn't cut it. I guess it could be replaced with a startup service but idk if that would work. You could try doing that with a similar approach as the default-flatpaks module and report the findings if interested.

Otherwise, this is a known limitation and there's not much else we can do about it.

**Comment by bsherman** [comment]
Well, I agree that stock sway itself does not automatically start desktop files, however... Fedora Sericea pre-configures sway to  start `*.desktop` apps in `$HOME/.config/autostart/` .

@qoijjj that said, i think you're asking for a feature, not reporting a bug, right?

**Comment by qoijjj** [comment]
@bsherman yeah it's either a request to extend the feature to work with sericea out of the box, or a report of a bug where the feature is broken on sericea :smile:

**Comment by xynydev** [comment]
> Fedora Sericea pre-configures sway to start *.desktop apps in $HOME/.config/autostart/.

Interesting. If that works, it should mean that it works normally. @qoijjj are you on Sericea? There could be something in some log file somewhere indicating a failure to start due to missing deps or something...

**Comment by Heus-Sueh** [comment]
I can confirm that Yafti doesn't start on Hyprland either

This is my repository in case anyone wants to investigate further
https://github.com/Heus-Sueh/Viper-OS
